backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/service/PPOrderCandidateCreateUpdateRequest.java

```diff
@@ -40,14 +43,16 @@
 import org.adempiere.mm.attributes.AttributeSetInstanceId;
 import org.adempiere.warehouse.WarehouseId;
 import org.eevolution.api.PPOrderCreateRequest;
+import org.eevolution.productioncandidate.model.PPOrderCandidateId;
 
 import javax.annotation.Nullable;
 import java.time.Instant;
 
 @Value
 @JsonDeserialize(builder = PPOrderCreateRequest.PPOrderCreateRequestBuilder.class)
-public class PPOrderCandidateCreateRequest
+public class PPOrderCandidateCreateUpdateRequest
 {
+	PPOrderCandidateId ppOrderCandidateId;
```

Comment:
`@Nullable`

Reply:
also pls set Nullable/NonNull for the other fields too

Reply:
added